action.yml

```
@@ -50,11 +50,17 @@ runs:
           echo "::error title=⛔ error hint::Support Linux Only"
           exit 1
       - name: Check Auth Info Empty
-        if: ${{ inputs.authkey == '' && (inputs['oauth-secret'] == '' || inputs.tags == '') }}
+        if: ${{ inputs.authkey == '' && inputs['oauth-secret'] == '' }}
         shell: bash
         run: |
           echo "::error title=⛔ error hint::OAuth identity empty, Maybe you need to populate it in the Secrets for your workflow, see more in https://docs.github.com/en/actions/security-guides/encrypted-secrets and https://tailscale.com/s/oauth-clients"
           exit 1
+      - name: Check Tags Provided
+        if: ${{ inputs.authkey == '' && inputs.tags == '' }}
+        shell: bash
+        run: |
+          echo "::error title=⛔ error hint::At least one ACL tag is required for nodes created by this Action. Ensure an appropriate tag exists in your ACL and provide it in your workflow. See more in https://tailscale.com/kb/1068/acl-tags/"
```

This error hint is accurate only if the action is authorized via Tailscale OAuth client--not if an [auth key is used](https://tailscale.com/kb/1276/tailscale-github-action/#using-an-auth-key) instead.

Validity of a tag, like the suitability of the set of tags provided via the sometimes-required `tags` input parameter, seems out of scope, here.  Validity of any given tag, as well as suitability of the set of tags provided, can be determined by the server behind the API this action uses, but not by the action itself.

I suggest wording the message so as to guide the user to providing input parameters which satisfy the requirements for this action as they are documented on [its GitHub Marketplace page](https://github.com/marketplace/actions/connect-tailscale?version=v2).

```suggestion
          echo "::error title=⛔ error hint:Use of this action's `oauth-secret` input parameter requires that parameter `tags` be provided, but it is empty. See https://tailscale.com/kb/1276/tailscale-github-action/#add-the-tailscale-github-action-to-a-workflow and `devices` scope in https://tailscale.com/kb/1215/oauth-clients/#scopes"
```